lib/config.js

```diff
@@ -21,12 +21,13 @@ config.express.session.resave = true;
 config.express.session.saveUninitialized = true;
 config.express.session.cookie = {};
 config.express.session.cookie.secure = true;
-// NOTE: 'connect' doesn't update the expires age for the cookie on every
-//   request so sessions will always timeout on the client after the maxAge
-//   time. Setting to null will cause sessions checks to only happen on the
-//   server which does update the expires time on every request. The server
-//   session defaultExpirationTime is set below.
+// NOTE: setting maxAge will allow cookies to persist
+// after the ttl has expired
 config.express.session.cookie.maxAge = null;
+// ttl is set in milliseconds
+// session stores such as MongoStore should use this.
```

Comment:
what does this mean?  I feel like more detail is needed here in order to be helpful.  Maybe some documentation about this new feature in a readme?

Reply:
we do need to resolve this. This expands on the explanation and adds a README section as session-storage implementations will need some instruction. https://github.com/digitalbazaar/bedrock-express/pull/21/commits/f1761713d345e634d4f13cd7c933bd796f7320b6